app/js/reducers/labOrderReducer.js

```diff
@@ -0,0 +1,18 @@
+import initialState from './initialState';
```

Comment:
My nitpick here would be shouldn't this really be labOrdersReducer.js ? :)

Reply:
yeah, it should, just noticed that! Wow